packages/api-bindings/src/graphql/utils/profile.ts

```diff
@@ -16,56 +17,71 @@ export type SupportedFollowModuleSettingsFragment =
 
 export type ProfileFieldsFragmentWithSupportedFollowModule = Overwrite<
   ProfileFieldsFragment,
-  { followModule: SupportedFollowModuleSettingsFragment }
+  { __followModule: SupportedFollowModuleSettingsFragment }
 >;
 
 export type ProfileFieldsFragmentWithRevertFollowModule = Overwrite<
   ProfileFieldsFragment,
-  { followModule: RevertFollowModuleSettingsFragment }
+  { __followModule: RevertFollowModuleSettingsFragment }
 >;
 
 export type ProfileFieldsFragmentWithFeeFollowModule = Overwrite<
   ProfileFieldsFragment,
-  { followModule: FeeFollowModuleSettingsFragment }
+  { __followModule: FeeFollowModuleSettingsFragment }
+>;
+
+export type ProfileFieldsFragmentWithUnknownFollowModule = Overwrite<
+  ProfileFieldsFragment,
+  { __followModule: UnknownFollowModuleSettingsFragment }
 >;
 
 export function isProfileFieldsFragmentWithSupportedFollowModule(
   profile: ProfileFieldsFragment,
 ): profile is ProfileFieldsFragmentWithSupportedFollowModule {
-  const followPolicyType = getFollowPolicyTypeFromProfileFieldsFragment(profile);
-  return followPolicyType !== null;
+  const followPolicyType = getFollowPolicyTypeFromProfileFieldsFragment(profile.__followModule);
+  return followPolicyType !== FollowPolicyType.UNKNOWN;
 }
```

Comment:
Is this helper used at all or just got migrated?

It seems if was used to guard UI from rendering in case of unknown follow module.

I think the new `followPolicy: FollowPolicy` edge is doing a better job. One just need to check `profile.followPolicy.type === FollowPolicyType.UNKNOWN`.